app/src/main/res/layout/cast_card.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="88dp"
```

Comment:
Maybe use WRAP_CONTANT, and fixed ImageView width?

Reply:
Homework №2:
"We follow best practices and thus all strings and colors should be in resources and not hardcoded in layout xml."

Reply:
> Maybe use WRAP_CONTANT, and fixed ImageView width?

I updated the layout, now this dimen is necessary


> Maybe dimensions should be in resources too?

Done